docker-compose.yml

```
@@ -88,6 +88,13 @@ services:
                     devices:
                     - driver: nvidia
                       capabilities: ["gpu", "utility", "compute"]
+    object-detection-azure:
+        image: ghcr.io/shared-reality-lab/image-preprocessor-object-detection-azure:unstable
+        labels:
+            ca.mcgill.a11y.image.preprocessor: 6
```

Run earlier, otherwise sorting/grouping won't apply.
```suggestion
            ca.mcgill.a11y.image.preprocessor: 3
```

---

But 3 is already assigned to the YOLO, does this have to run before grouping and sorting ? in that case it would be better to change their numbers.

---

(for example, OCR and autour are both labeled `2` since they can run in parallel - they are completely independent of each-other.)